src/elements/simple-input/src/index.tsx

```diff
@@ -22,14 +26,38 @@ export const SimpleInput = React.forwardRef(
       onChange,
       defaultValue,
       prefix,
-      suffix
+      suffix,
+      inputSize
     } = props
 
     const [hasFocus, setHasFocus] = useState(false)
+    const [prefixWidth, setPrefixWidth] = useState(0)
+    const [suffixWidth, setSuffixWidth] = useState(0)
+    const prefixRef: RefObject<HTMLDivElement> = React.createRef()
+    const suffixRef: RefObject<HTMLDivElement> = React.createRef()
+
+    useEffect(() => {
+      setPrefixWidth(prefixRef?.current?.offsetWidth || 0)
+      setSuffixWidth(suffixRef?.current?.offsetWidth || 0)
+    }, [suffixRef])
```

Comment:
you should probably add prefixRef here too `[prefixRef, suffixRef]`

Reply:
Good catch, thanks @Segger87 - fixed it!